config.ini.example

```diff
@@ -25,6 +25,9 @@ domain=localhost:5080
 # Change this value to something random and secret
 secret-key=hello world
 
+# Set this to the id of a Game that should automatically sync up with CKAN's KSP builds.json
+ksp-game-id=
```

Comment:
The description of this config key doesn't really match the key name.
> the id of **a** Game
> **ksp**-game-id

I think I would write it clearly as "the id of **KSP**", since the key itself already has `ksp` in its name.

Reply:
Well, I want to emphasize "Game" to make it clear which table this is is from. KSP is mentioned later in the description.
Maybe a two-line description would be better...

Reply:
That works better, thanks.